### PR TITLE
Feat:Signup form added

### PR DESCRIPTION
**Pull request by kotalwar-krishna**
Signup form related Html you can find in sign-up.html
sign up related css you find i signup.css file.
login box attached to  sign in button

**screenshots**
img7
img8

### CODE REVIEW

**Review by Kothalkarpriya — APPROVED** [review]
Remove extra space, and you are good to go

File: signup.css [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+}
+  
+.SignUp-banner-heading h1{
+      color:var(--black);
+      font-size: 4rem;
+}
+
+.SignUp-banner-heading h1 span{
+    color: var(--orange);
+}
```
Comment (sks-21) [inline signup.css]: be consistent with naming. use kebab case only, don't mix camel case

File: signup.css [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  }
+
+  .signup-form-container .container hr {
+    border: 1px solid #0f0f0f;
+    margin-bottom: 3rem;
+  }
+
+  .signin {
+    background-color: var(--light-color-two);
+    text-align: center;
```
Comment (sks-21) [inline signup.css:54]: you can have better color names

File: sign-up.html [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+<!-- Sign up Section start -->
+<section class="signup-form-container">
+    <form>
+        <div class="container">
+          <h1>Register</h1>
+          <p>Please fill in this form to create an account.</p>
+          <hr>
+
+          <label for="user-name"><b>Name</b></label>
+          <input type="text" placeholder="Enter youe Name" id="user-name">
```
Comment (sks-21) [inline sign-up.html]: typo in 'your'

**Review by sks-21 — APPROVED** [review]
Suggested some changes. Rest everything looks good.

**Review by sakleprajwal — APPROVED** [review]
Looks good to me. Please go ahead

File: signup.css (hunk 1 above)
Comment (kotalwar-krishna) [reply signup.css]: ok

File: signup.css (hunk 2 above)
Comment (kotalwar-krishna) [reply signup.css:54]: ya I know, but at the time of writing i am not able better color name. i will do this

File: sign-up.html (hunk 3 above)
Comment (kotalwar-krishna) [reply sign-up.html]: coorected

**Comment by psychoactiv** [comment]
looks good to me

**Pull request merged**